statistics/handle/update.go

```diff
@@ -224,19 +236,31 @@ func (s *SessionStatsCollector) StoreQueryFeedback(feedback interface{}, h *Hand
 func (s *SessionStatsCollector) UpdateColStatsUsage(colMap colStatsUsageMap) {
 	s.Lock()
 	defer s.Unlock()
-	s.colMap.merge(colMap)
+	newMap := make(colStatsUsageMap)
```

Comment:
`colStatsUSage` record which columns' stats are used by query optimization. It doesn't affect table stats.

Reply:
ok. will remove it.